hadoop-hdds/docs/content/concept/_index.zh.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Concepts
```

Comment:
I have replaced the content with Chinese on my laptop and it seems to work out just fine. Except that the language switch button is not exactly on the top right, I am not sure if it's a problem. Screenshot:
![](https://tva1.sinaimg.cn/large/006tNbRwly1g9vvw3g5ttj31ip078q42.jpg)
on the other hand, index page seems to be okay on this subject:
![](https://tva1.sinaimg.cn/large/006tNbRwly1g9vvxeceoej31hc05lt9j.jpg)

Reply:
I think it is on the right top of the content frame.